### DISCUSSION TITLE
GTR Mini in Simulator

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#150] [Developer Feedback]
I noticed you guys just added GTR Mini system image and this is cool, but it is not usable.
Once I try to install any of my watchfaces on it, it freezes. I believe it is caused by new (output) image format you use in mini. 
I look forward for updated tools such as zeus/vsc/simulator, so we could use it.
@MyoungXUE

**Comment by MyoungXUE** [comment]
Hello, have you updated the simulator and the Zeus CLI to the latest version?

**Reply by zdunex25** [reply]
Both Zeus and Simulator has been updated in late 2022. So according to this I am up to date: https://docs.zepp.com/docs/guides/tools/cli/release-note/

**Reply by zdunex25** [reply]
@MyoungXUE new system image (gtr mini v1.0.1) kinda works, i can install apps and watchfaces, but screen is square not circular 
![image](https://user-images.githubusercontent.com/1674725/234249571-5008bbc2-cac7-4048-bf88-f54325b5c5fc.png)

system image size matches gtr 4 mini image size, so it has to be the case
